examples/animation/double_pendulum.py

```diff
@@ -14,13 +14,15 @@
 import matplotlib.pyplot as plt
 import scipy.integrate as integrate
 import matplotlib.animation as animation
+from collections import deque
 
 G = 9.8  # acceleration due to gravity, in m/s^2
 L1 = 1.0  # length of pendulum 1 in m
 L2 = 1.0  # length of pendulum 2 in m
 M1 = 1.0  # mass of pendulum 1 in kg
 M2 = 1.0  # mass of pendulum 2 in kg
-t_stop = 5  # how many seconds to simulate
+t_stop = 50  # how many seconds to simulate
```

Comment:
It may be that the time was intentionally set to a not too high value. The animation is embedded in the example and also as a gif in the example overview page. It should be checked that these do not grow too much in size.

Reply:
I was curious to see the behavior of the pendulum for longer times. But I agree that as an example, t=5 is just fine. Changing back to t=5.

Thanks for the review! Your suggestions were indeed helpful!